### DISCUSSION TITLE
HTTP Streaming Will Not Turn Off

### DISCUSSION THREAD

**Discussion by R101G** [rern/rAudio#1805] [Bug Reports]
This feature turns on at every restart. Is there a reason, or is this a bug?

**Comment by markblinkhorn** [comment]
FWIW - I don't have this issue on any of my installations. I have never enabled it and it remains off. You may find some clues here: [https://wiki.archlinux.org/title/Music_Player_Daemon/Tips_and_tricks#HTTP_streaming](url) 

For reference, my /etc/mpd.conf looks like this:
```
rAudio3:~  #  cat /etc/mpd.conf
# See: /usr/share/doc/mpd/mpdconf.example

pid_file "/run/mpd/mpd.pid"
db_file "/var/lib/mpd/mpd.db"
state_file "/var/lib/mpd/mpdstate"
playlist_directory "/var/lib/mpd/playlists"
rAudio3:~  #

```

**Comment by rern** [comment]
When there was no sound card, `httpd` will be enabled as an output instead. MPD won't work properly without one.

**Reply by R101G** [reply]
@rern 
So if I unplug the usb sound card, this feature gets enabled?
Would it be possible to have it turn off automatically too, unless it is actually selected by the user?

**Comment by Sagnasty** [comment]
I would also prefer to have manual control over this feature.

Automatically enabling HTTP streaming creates some headaches when I put my USB DAC  into standby mode (no USB communication).

Jim

Raspberry Pi 5B Rev 1.0
Raspberry Pi 3B+ Rev 1.3 > HiFiBerry DAC+ Pro
Synology DS718+
